MagaluCloud/magalu
# Add pre commit hooks with Go and Py linters

## Description

Add commit hooks for the repository using `pre-commit` tool. For now, the tool will check for:

* `golangci-lint` running multiple linter on staged Go files
* `flake8` for Python files
* `commitzen` (Conventional Commits) on all commit messages

## Related Issues

- Closes #1 

## How to test it

1. Add or modify a Go file that does not conform with one of the linters ran by `golangci-lint`
2. Commit and check for errors

^ Repeat the above for Python files and for commit messages that does not obey conventional commit guidelines
